### PR TITLE
Add I_love_music.md

### PR DESCRIPTION
**Pull request by dzxxbj**
Please review this, @casey-collab

### CODE REVIEW

**Comment by casey-collab** [comment]
Unfortunately this pull request could not be automatically reviewed due to technical difficulty.

**Comment by cbuckey-uda** [comment]
@casey-collab, please try again

File: Chinese/I_love_music/I_love_music.md [hunk 1]
```diff
@@ -0,0 +1 @@
+I love music!
```
Comment (casey-collab) [inline Chinese/I_love_music/I_love_music.md]: It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by cbuckey-uda** [comment]
Closing this pull request due to lack of activity. If you ever have time to respond to the comments above, feel free to re-open it.

**Pull request closed without merging**